spec/em-eventsource_spec.rb

```diff
@@ -47,6 +47,13 @@ def create_response_headers(status, content_type="", other={})
     end
   end
 
+  it "connect to the good server with specified method" do
+    start_source "http://example.net/streaming", {}, {}, :post do |source, req|
+      req.method.must_equal :post
```

Comment:
Does the more general "connects to the good server" test assert that it's a GET request? It's unclear whether `get_args` would only return the GET arguments (which would be fine to assert that it's a GET), or whether it's _getting_ the arguments (which would mean we might want to add an explicit assertion to ensure it defaults to GET).